### PR TITLE
Patch release 2.7.1

### PR DESCRIPTION
**Pull request by sateeshperi**
patch release to better support cloud execution. This patch fixes the symlink creation of fasta in index folder

### CODE REVIEW

File: tests/bismark_nomeseq.nf.test.snap [hunk 1]
```diff
@@ -41,20 +41,6 @@
...
-                "bismark/coverage2cytosine/reports/SRR389222_sub1.NOMe.GpC_report.txt.gz",
-                "bismark/coverage2cytosine/reports/SRR389222_sub2.NOMe.CpG_report.txt.gz",
-                "bismark/coverage2cytosine/reports/SRR389222_sub2.NOMe.GpC_report.txt.gz",
-                "bismark/coverage2cytosine/reports/SRR389222_sub3.NOMe.CpG_report.txt.gz",
-                "bismark/coverage2cytosine/reports/SRR389222_sub3.NOMe.GpC_report.txt.gz",
-                "bismark/coverage2cytosine/summaries",
-                "bismark/coverage2cytosine/summaries/Ecoli_10K_methylated.cytosine_context_summary.txt",
-                "bismark/coverage2cytosine/summaries/SRR389222_sub1.cytosine_context_summary.txt",
-                "bismark/coverage2cytosine/summaries/SRR389222_sub2.cytosine_context_summary.txt",
-                "bismark/coverage2cytosine/summaries/SRR389222_sub3.cytosine_context_summary.txt",
```
Comment (maxulysse) [inline tests/bismark_nomeseq.nf.test.snap:57]: lots of deletion here

File: tests/bismark_nomeseq.nf.test.snap [hunk 2]
```diff
@@ -507,18 +493,6 @@
...
-                "SRR389222_sub1.NOMe.CpG_report.txt.gz:md5,731e940264ca803ca48842a1fc9402ad",
-                "SRR389222_sub1.NOMe.GpC_report.txt.gz:md5,96c3f1c2f3d334baafb55f0805eaf52c",
-                "SRR389222_sub2.NOMe.CpG_report.txt.gz:md5,2ec3a32d0e8239ba50c079e1519aaf00",
-                "SRR389222_sub2.NOMe.GpC_report.txt.gz:md5,bcde6f7e90831604368ae6430e5695f0",
-                "SRR389222_sub3.NOMe.CpG_report.txt.gz:md5,f9793efc46ecdf413a84338e9f9f42dd",
-                "SRR389222_sub3.NOMe.GpC_report.txt.gz:md5,e4ab9e9c4d4ea2fe3ca2378f3802f6f6",
-                "Ecoli_10K_methylated.cytosine_context_summary.txt:md5,482eac2bc68ca28545bab71c0c414c3a",
-                "SRR389222_sub1.cytosine_context_summary.txt:md5,1d4c33ccf5e249907ca2ba50a37c7697",
-                "SRR389222_sub2.cytosine_context_summary.txt:md5,8f7684582aa75b24197866138e6d1a4c",
-                "SRR389222_sub3.cytosine_context_summary.txt:md5,7ba71718c6556e8b3019e1704948d562",
```
Comment (maxulysse) [inline tests/bismark_nomeseq.nf.test.snap:521]: and here

File: modules/nf-core/trimgalore/main.nf [hunk 3]
```diff
@@ -4,8 +4,8 @@ process TRIMGALORE {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/trim-galore:0.6.10--hdfd78af_0' :
-        'biocontainers/trim-galore:0.6.10--hdfd78af_0' }"
+        'https://depot.galaxyproject.org/singularity/trim-galore%3A0.6.10--hdfd78af_1' :
```
Comment (maxulysse) [inline modules/nf-core/trimgalore/main.nf:7]:
```suggestion
        'https://depot.galaxyproject.org/singularity/trim-galore:0.6.10--hdfd78af_1' :
```
why this change?

Comment (sateeshperi) [reply modules/nf-core/trimgalore/main.nf:7]: coming from [here](https://github.com/nf-core/modules/commit/8c5eeedd45e295fc9a4f164631da6a8b37e6b9c6)

File: tests/bismark_nomeseq.nf.test.snap (hunk 1 above)
Comment (sateeshperi) [reply tests/bismark_nomeseq.nf.test.snap]: Thanks for catching this. the module wasn't publishing that's y. And I changed that based on the vscode nextflow extension. reverted back now. should be all good

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,9 +1,12 @@
 # nf-core/methylseq
 
-## [v2.8.0](https://github.com/nf-core/methylseq/releases/tag/2.8.0) - []
+## [v2.7.1](https://github.com/nf-core/methylseq/releases/tag/2.7.1) - [2024-10-27]
 
 ### Bug fixes & refactoring
 
+- 🐛 fix fasta symlink in bismark & bwameth align modules [#448](https://github.com/nf-core/methylseq/pull/448)
+- 🐛 apply few new vscode nextflow extension [#448](https://github.com/nf-core/methylseq/pull/448)
```
Comment (mashehu) [inline CHANGELOG.md:8]: 🤫

**Review by mashehu — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (sateeshperi) [reply CHANGELOG.md:8]: I meant suggestions 😬

Comment (sateeshperi) [reply CHANGELOG.md:8]: will fix soon
